Toolchain/CMakeLists.txt

```
@@ -0,0 +1,101 @@
+cmake_minimum_required (VERSION 3.0)
+project (SerenityOSToolchain NONE)
+
+include(ExternalProject)
+
+include(ProcessorCount)
+ProcessorCount(MAKE_JOBS)
+
+find_program(MAKE_EXE NAMES gmake make)
+
+set(PREFIX ${CMAKE_CURRENT_SOURCE_DIR}/Local)
+set(TARGET i686-pc-serenity)
+set(SYSROOT ${CMAKE_CURRENT_SOURCE_DIR}/../Root)
+set(PATCHES ${CMAKE_CURRENT_SOURCE_DIR}/Patches)
+
+set(LIBC ${CMAKE_CURRENT_SOURCE_DIR}/../Libraries/LibC)
+set(LIBM ${CMAKE_CURRENT_SOURCE_DIR}/../Libraries/LibM)
+
+ExternalProject_Add(binutils
+    URL      "http://ftp.gnu.org/gnu/binutils/binutils-2.33.1.tar.gz"
```

Reason not to use https?

```suggestion
    URL      "https://ftp.gnu.org/gnu/binutils/binutils-2.33.1.tar.gz"
```